docs/migrations/v7-to-v8.md

```diff
@@ -80,7 +80,7 @@ TODO: https://github.com/cosmos/ibc-go/pull/3303
 
 ## Relayers
 
-- No relevant changes were made in this release.
+`channeltypes.EventTypeTimeoutPacketOnClose` (where `channeltypes` is an import alias for `"github.com/cosmos/ibc-go/v8/modules/core/04-channel"`) has been removed, since core IBC does not emit any event with this key.
```

Comment:
is there any reason why we don't? Should we be emitting it?

Reply:
It could be used [here](https://github.com/cosmos/ibc-go/blob/cf519bc4d4285aa0b69511ba08d69b7456744f5d/modules/core/04-channel/keeper/timeout.go#L227), but at the end of the day we are timing out a packet, with the only difference that the channel is already closed. Another place where it could be used is [here](https://github.com/cosmos/ibc-go/blob/cf519bc4d4285aa0b69511ba08d69b7456744f5d/modules/core/04-channel/keeper/timeout.go#L171), but `TimeoutExecuted` is called both in `Timeout` and `TimeoutOnClose` in the core message server.